PoGo.NecroBot.Logic/Tasks/UseIncubatorsTask.cs

```diff
@@ -12,6 +17,8 @@ namespace PoGo.NecroBot.Logic.Tasks
 {
     internal class UseIncubatorsTask
     {
+        private static readonly string IncubatorUsagePath = Path.Combine(Directory.GetCurrentDirectory(), "Temp", "Incubators.json");
```

Comment:
I suspect this will break with multiple profiles.
private static readonly string IncubatorUsagePath = Path.Combine(Directory.GetCurrentDirectory(), "Temp", "Incubators.json");